plugins/main/bridge/bridge.go

```diff
@@ -151,9 +160,23 @@ func setupVeth(netns string, br *netlink.Bridge, ifName string, mtu int) error {
 		return fmt.Errorf("failed to connect %q to bridge %v: %v", hostVethName, br.Attrs().Name, err)
 	}
 
+	// set hairpin mode if enabled
+	if hairpinMode {
```

Comment:
Can't we just call `netlink.LinkSetHairpin(hostveth)` here?
https://godoc.org/github.com/vishvananda/netlink#LinkSetHairpin